guides/telemetry.md

```diff
@@ -0,0 +1,86 @@
+# Telemetry
+
+Absinthe 1.5 supports your use of `:telemetry` to instrument its activity.
+Call `:telemetry.attach/4` or `:telemetry.attach_many/4` to attach your
+handler function to any of the following event names:
+
+- `[:absinthe, :execute, :operation, :start]` when the operation starts
+- `[:absinthe, :execute, :operation]` when the operation finishes
+- `[:absinthe, :subscription, :publish, :start]` when a subscription starts
+- `[:absinthe, :subscription, :publish]` when a subscription finishes
+- `[:absinthe, :resolve, :field, :start]` when field resolution starts
+- `[:absinthe, :resolve, :field]` when field resolution finishes
+
+Only fields with a resolver get measured.
+
+For async, batch, and dataloader fields, Absinthe sends the final event when
+it gets the results. That might be later than when the results are ready. If
+you need to know how long the underlying operation took, you'll need to hook
+telemetry up to that underlying operation. See, for example, the recommended
+telemetry events in the documentaiton for `Ecto.Repo`.
+
+## Measurements
+
+- `start_time`: sent with event names ending `:start`
```

Comment:
`ending` should maybe be `ending with` right?

Reply:
Yep.